### PR TITLE
🐛 Bug fix allowing set height and fixing Ad position

### PR DESCRIPTION
**Pull request by rochapablo**
1. The right min-height for StickAd it's 50 and not 100 as it's hard-code.
2. Removing some unused attributes

### CODE REVIEW

**Review by powerivq — APPROVED** [review]

File: extensions/amp-auto-ads/0.1/anchor-ad-strategy.js [hunk 1]
```diff
@@ -87,13 +87,14 @@ export class AnchorAdStrategy {
...
+    const baseAttributes = this.baseAttributes_;
     const viewportWidth = Services.viewportForDoc(this.ampdoc).getWidth();
     const attributes = /** @type {!JsonObject} */ (Object.assign(
       dict(),
-      this.baseAttributes_,
+      baseAttributes,
       dict({
         'width': String(viewportWidth),
-        'height': '100',
+        'height': baseAttributes.height || '100',
```
Comment (powerivq) [inline extensions/amp-auto-ads/0.1/anchor-ad-strategy.js:97]: Why is this helpful? I see neither you or other providers are setting the `height` attribute for sticky ads.

Comment (rochapablo) [reply extensions/amp-auto-ads/0.1/anchor-ad-strategy.js:97]: `baseAttributes.height` allow me to set max-height from `getConfigUrl()` JSON response.

**Comment by powerivq** [comment]
Please fix the merge conflict.

**Comment by rochapablo** [comment]
@powerivq, conflict fixed.

**Comment by rochapablo** [comment]
@powerivq, can you review the PR please?

**Comment by powerivq** [comment]
Look like there's another conflict still here.

**Comment by rochapablo** [comment]
@powerivq, can you review the PR please?

**Review by powerivq — APPROVED** [review]

**Pull request merged**